src/main/java/com/nekomaster1000/infernalexp/entities/WarpbeetleEntity.java

```diff
@@ -160,6 +151,11 @@ public void livingTick() {
             if (this.isConverting() && this.conversionTicks > 0) {
                 this.conversionTicks--;
                 if (this.conversionTicks == 0) {
+                    if (!this.isChorus()) {
+                        this.setChorus(true);
+                    } else {
+                        this.setChorus(false);
+                    }
```

Comment:
```suggestion
                    this.setChorus(!this.isChorus());
```
I think this might be just a bit cleaner? Up to Swan

Reply:
I approve this :+1: